docs/manifest.json

```diff
@@ -137,7 +137,7 @@
             },
             {
               "title": "API Middlewares",
-              "path": "/docs/api-routes/api-middlewares.md"
```

Comment:
Do we want to add a redirect for the old path?

Reply:
https://github.com/vercel/front/pull/15506

Reply:
Oh yeah, I forgot you can add it in the manifest now.